src/fieldlayoutelements/PurchasablePromotableField.php

```diff
@@ -54,11 +59,23 @@ public function inputHtml(ElementInterface $element = null, bool $static = false
             'id' => 'promotable',
             'name' => 'promotable',
             'small' => true,
-            'on' => $element->promotable,
+            'on' => $element->getIsDraft() ? $this->defaultPromotable : $element->promotable,
```

Comment:
Not sure if `getIsDraft` is enough to know if its the first time viewing the element. Anything else we could check to know?

Reply:
Changed it to `.isFresh()`